script/Ciel/Day 15/15_01A_CIEL15A_1.txt

```diff
@@ -1793,7 +1799,8 @@ Our house is actually a small bakery, and you've even been there shopping, remem
 [sha:dbc9d39b3f08b97ebd66df6ea7572ee2ec54d727]{
 -- Page 111, Offset 41711.
 -- ……ああ。そういう事も、ありえたのかもしれない。
-Yeah... I suppose that could have happened, someplace, somewhere.
+Yeah... Maybe there was a world in which that happened.
+Yeah... Maybe that is something that could have happened.
```

Comment:
don't really like the double 'yeah...', first one is enough imo

Reply:
This was just me forgetting to go back and clean things up. This and below are two versions of the same line.

Still don't love the someplace/somewhere, but I get the link it was going for. Maybe
> Yeah... That does sound like the kind of thing that could've happened.
> But it didn't.

Reply:
Sounds a bit banal

prefer
>Yeah... maybe somewhere out there that could have happened.

>But it didn't